src/schemas/json/sublime-syntax.json

```
@@ -0,0 +1,248 @@
+{
+  "$comment": "https://www.sublimetext.com/docs/syntax.html",
+  "$id": "https://json.schemastore.org/sublime-syntax.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "meta_pattern": {
+      "type": "object",
+      "properties": {
+        "meta_scope": {
+          "description": "This assigns the given scope to all text within this context, including the patterns that push the context onto the stack and pop it off.",
```

You can make short explanations but provide links to docs via this format:

```suggestion
          "description": "TEXT\nURL",
```